config/country_dialing_codes.yml

```diff
@@ -1,5 +1,3 @@
-# NOTE: if updating this to match current AWS config, consider bringing back
-# the daily CircleCI check added in 3d4194c1f76ca1554852bacfde4abdea2ebb6104
```

Comment:
Maybe we should bring back the `.circleci/config.yml` changes? And then just have the script do `|| true` and put some comment there instead?

Reply:
I brought it back in 328ef30 and disabled it via 1964223. How does that look to you? 

I assume the idea is to have it easily re-enabled when the current diffs are resolved? I also think it'd be good to avoid booting up the container at all.